browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -1045,6 +1049,10 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 			this._update();
 			this.enableDrawing();
 		}
+        if(this.isHighlightEnabled)
+            L.CanvasTileLayer.prototype._highlightColAndRow.call(this, textMsg, this._selectedPart);
```

Comment:
why not `this._highlightColAndRow(textMsg)` ?
`this._selectedPart` doesn't have to be passed as you can read it inside

Reply:
can't we do this?

Reply:
line can be simplified:
`L.CanvasTileLayer.prototype._highlightColAndRow.call(this, textMsg, this._selectedPart);`
to
`this._highlightColAndRow(this, textMsg);`

and inside `_highlightColAndRow` function you can get your `this._selectedPart` without passing as argument